pkg/kwokctl/runtime/kind/cluster.go

```diff
@@ -199,9 +199,8 @@ func (c *Cluster) Up(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
-	err = c.Kubectl(ctx, utils.IOStreams{
-		ErrOut: os.Stderr,
-	}, "apply", "-f", utils.PathJoin(conf.Workdir, runtime.KwokDeploy))
+
+	err = utils.Exec(ctx, "", utils.IOStreams{}, "docker", "cp", utils.PathJoin(conf.Workdir, runtime.KwokPod), conf.Name+"-control-plane:/etc/kubernetes/manifests/")
```

Comment:
https://github.com/kubernetes-sigs/kwok/blob/c6ae593ff5c39497ad44c0bcef09431ca6bd9421/pkg/kwokctl/runtime/kind/cluster.go#L276-L298

It looks like Stop and Start cannot operate on the Kwok-controller.

```suggestion
	err = utils.Exec(ctx, "", utils.IOStreams{}, "docker", "cp", utils.PathJoin(conf.Workdir, runtime.KwokPod), conf.Name+"-control-plane:/etc/kubernetes/manifests/kwok-controller.yaml")
```

Or modify `runtime.KwokPod`

Reply:
I agree with you kwok-controller.yaml,modified

Reply:
@wzshiming Start and Stop In this mode, whether to stop when moving to another folder?

Reply:
> Start and Stop In this mode, whether to stop when moving to another folder?

Yes